pyfar/constants/constants.py

```diff
@@ -433,3 +434,149 @@ def density_of_air(
+    10 Hz to 20 kHz.
+
+    Parameters
+    ----------
+    num_fractions : 1, 3
+        The number of octave fractions. ``1`` returns octave center
+        frequencies, ``3`` returns third octave center frequencies.
+    frequency_range : array, tuple
+        The lower and upper frequency limits, the default is
+        ``(20, 20e3)``.
+        ``(20, 20e3)`` is in accordance with IEC 61260-1
+        ``(10, 20e3)`` is in accordance with IEC 61672-1 [#]_
+
+    Returns
+    -------
+    nominal : numpy.ndarray of float
+        The nominal center frequencies.
+
+    References
+    ----------
+    .. [#] International Electrotechnical Commission, "IEC 61260-1:2014 -
+        Electroacoustics - Octave-band and fractional-octave-band filters -
+        Part 1: Specifications", IEC, 2014.
+
+    .. [#] International Electrotechnical Commission,
+        "IEC 61672-1:2013 - Electroacoustics - Sound level meters - Part 1:
+        Specifications", IEC, 2013.
+    """
+    G = 10**(3/10)
+    f_lims = np.asarray(frequency_range)
+    if f_lims.size != 2:
+        raise ValueError(
+            "You need to specify a lower and upper limit frequency.")
+    if f_lims[0] > f_lims[1]:
+        raise ValueError(
+            "The second frequency needs to be higher than the first.")
+    #
+    if num_fractions == 1:
+            if (f_lims[0] > 15.8*G**(-1/2)) and ( f_lims[1] <
+                                                 15848.93192*G**(1/2)):
+                nominal = np.array([
+                    16, 31.5, 63, 125, 250, 500, 1e3,
+                    2e3, 4e3, 8e3, 16e3], dtype=float)
+            else:
+                warnings.warn(
+                "The nominal center frequencies for one-third-octave-band " \
+                "are defined only from 11.2 Hz to 22387.2 Hz.",  stacklevel=2)
+    elif num_fractions == 3:
+        if (f_lims[0] > 10*G**(-1/6)) and (f_lims[1] < 19952.62315*G**(1/6)):
+            nominal = np.array([
+                10, 12.5, 16, 20, 25, 31.5, 40, 50, 63, 80,
+                100, 125, 160, 200, 250, 315, 400, 500, 630,
+                800, 1000, 1250, 1600, 2000, 2500, 3150, 4000,
+                5000, 6300, 8000, 10000, 12500, 16000, 20000], dtype=float)
+        else:
+            warnings.warn(
+                "The nominal center frequencies for octave-band filter are " \
+                "defined only from 8.91 Hz to 22387.2 Hz.",  stacklevel=2)
+
+    mask = (nominal >= f_lims[0]) & (nominal <= f_lims[1])
+    nominal = nominal[mask]
+    return nominal
+
+def fractional_octave_frequencies_exact(
+        num_fractions=1, frequency_range=(20, 20e3)):
+    r"""Return the exact center frequencies for fractional-octave-band filters.
+
+    The frequencies are calculated in accordance with the IEC 61260-1:2014
+    standard [#]_.
+
+    The octave frequency ratio, :math:`G`, is given by the following
+    expression.
+
+    .. math::
+
+        G = 10^{3/10} \approx 1.99526
```

Comment:
```suggestion
        G = 10^{3/10}
```
not sure if we need it, because we dont use 1.999.. but directly 10^0.3